docs/rst/installation/linux_installation.rst

```diff
@@ -2,3 +2,292 @@
+It is required to build the :ref:`fastdds_qos_profiles_manager_lib_api_reference` from the header files.
+In Ubuntu, the dependency can be installed running:
+
+.. code-block:: bash
+
+    sudo apt install doxygen
+
+Python3 virtual environment (recomended)
+""""""""""""""""""""""""""""""""""""""""
+
+It is recommended to install the *Documentation* tool dependencies in a `python3 <https://www.python.org/downloads/>`_ virtual environment.
+That would avoid polluting the user's installation setup.
+In Ubuntu, python3 can be installed running:
+
+.. code-block:: bash
+
+    sudo apt install python3
+
+Tool suite build process
+------------------------
+
+Create a workspace directory and download the project:
+
+.. code-block:: bash
+
+    mkdir -p ~/fastdds_qosprofman_ws/src
+    cd ~/fastdds_qosprofman_ws/src
+    git clone https://github.com/eProsima/Fast-DDS-QoS-Profiles-Manager.git
+
+.. note::
+
+    It is recommended to deploy the project in a python3 virtual environment, by running the following commands:
+
+    .. code-block:: bash
+
+        cd ~/fastdds_qosprofman_ws
+        python3 -m venv fastdds_qosprofman_venv
+        source fastdds_qosprofman_venv/bin/activate
+
+The *Documentation* tool has several dependencies described in the ``requirements.txt`` project file.
+
+.. literalinclude:: /requirements.txt
+    :language: text
+
+Install them using `pip3 <https://docs.python.org/3/installing/index.html>`_:
+
+.. code-block:: bash
+
+    pip3 install -r ~/fastdds_qosprofman_ws/src/Fast-DDS-QoS-Profiles-Manager/docs/requirements.txt
+
+.. _installation_build_source_linux:
+
+.. tabs::
+
+    .. tab:: Building with colcon
+
+        *eProsima Fast DDS QoS Profiles Manager* complete tool suite is build using `colcon <https://colcon.readthedocs.io/en/released/>`_  by running:
+
+        .. code-block:: bash
+
+            cd ~/fastdds_qosprofman_ws
+            colcon build
+
+        .. note::
+
+            Being based on `CMake <https://cmake.org/>`_ it is possible to pass `CMake configuration options <https://colcon.readthedocs.io/en/released/reference/verb/build.html#cmake-specific-arguments>`_ to the :code:`colcon build` command.
+            These options can be also passed through a `colcon.meta <https://colcon.readthedocs.io/en/released/user/configuration.html?highlight=colcon.meta#using-meta-files>`_ file.
+
+            For instance, the *Library* test suite building is enabled by building with the CMake option `EPROSIMA_BUILD_TESTS`, among adding the GTest dependency (``"googletest-distribution"``) in the *Library* `colcon.pkg <https://colcon.readthedocs.io/en/released/user/configuration.html?highlight=colcon.meta#colcon-pkg-files>`_ file. Also, colcon provides `certain arguments <https://colcon.readthedocs.io/en/released/reference/package-selection-arguments.html>`_ to perform package selection, excluding undesired packages or including specific packages in the build process.
```

Comment:
Adding GTest to the colcon.pkg is only required if the GTest repo has been cloned into the workspace.